test/google_scraping/dashboard/keyword_scarper_worker_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule GoogleScraping.Dashboard.KeywordScraperWorkerTest do
```

Comment:
Shall we move this to a `google_scraping_worker` folder so it mirrors the application folder structure? 😄

Reply:
Thanks, it was fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/41/commits/03c2b57cd07d71175fed403065e39d1846131ed4